frontend/lib/resolution/resolution-queries.cpp

```diff
@@ -3956,6 +4024,142 @@ bool isTypeDefaultInitializable(Context* context, const Type* t) {
   return isTypeDefaultInitializableQuery(context, t);
 }
 
+void getCopyOrAssignableInfo(Context* context, const Type* t,
+                                    bool& fromConst, bool& fromRef,
+                                    bool checkCopyable);
+
+// Determine whether a class type is copyable or assignable, from ref and/or
+// from const.
+// Return: First is from const, second is from ref.
```

Comment:
Probably best to just remove this comment now that you aren't using a tuple anymore